ci-lua

```diff
@@ -1,25 +1,43 @@
-#!/bin/sh
+#!/bin/bash
 
 . ./config
 
-rm -rf "$TEST_OUT_DIR/lua"
-mkdir -p "$TEST_OUT_DIR/lua"
+ALLOWED_LUA_VERSIONS=("5.1" "5.2" "5.3")
+CURRENT_LUA_VERSION=""
+
+# try to use $VARIETY from env
+for allowed_version in "${ALLOWED_LUA_VERSIONS[@]}"; do
+  if [[ "${allowed_version}" = "${VARIETY}" ]]; then
+    CURRENT_LUA_VERSION="${VARIETY}"
+  fi
+done
```

Comment:
You don't really need bash, arrays and [double brackets](https://www.tldp.org/LDP/abs/html/testconstructs.html#DBLBRACKETS) here. Just a normal sh iteration is totally fine:

```suggestion
ALLOWED_LUA_VERSIONS='5.1 5.2 5.3'
CURRENT_LUA_VERSION=

# try to use $VARIETY from env
for allowed_version in $ALLOWED_LUA_VERSIONS; do
	if [ "${allowed_version}" = "${VARIETY}" ]; then
		CURRENT_LUA_VERSION="${VARIETY}"
	fi
done
```

Also please keep the formatting consistent: this file uses tabs, not spaces.